src/building/how-to-build-and-run.md

```diff
@@ -152,7 +152,9 @@ is what you need to build other Rust programs (unless you use `#![no_std]` or
 
 You will probably find that building the stage1 `std` is a bottleneck for you,
 but fear not, there is a (hacky) workaround...
-see [the section on "recommended workflows"](./suggested.md) below.
+see [the section on "recommended workflows"][keep-stage] below.
```

Comment:
should *below* be here

Reply:
Also might be better to say "see the section on incremental builds" (or something similar) since the link target changed.

Reply:
"incremental" is very misleading here, it's not the same as `-C incremental`.

I agree "below" is outdated and should be removed.